src/test/java/seedu/recipe/storage/jsonadapters/JsonAdaptedTimeUnitTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.recipe.storage.jsonadapters;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+//import static seedu.recipe.testutil.Assert.assertThrows;
+import static seedu.recipe.testutil.TypicalRecipes.CACIO_E_PEPE;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.model.recipe.unit.TimeUnit;
+
+public class JsonAdaptedTimeUnitTest {
+    private static final String VALID_UNIT = "hour";
+    private static final String INVALID_UNIT = "123";
+
```

Comment:
Does this cover all equivalence partitions?

Reply:
This should be done in model tests.